testCases/testAll.sh

```diff
@@ -42,12 +42,14 @@ do
 
     runCallBack "./tearDown.sh"
 
+    testName=$(echo $testCase | sed -e 's/\([A-Z]\)/ \1/g')
```

Comment:
nonono... i see you intend here, but now test names do not correspond with directory.
also it will be cool to not mix tester alter and code alter in one PR imho :)

Reply:
fixed